docs/en/database_engines/mysql.md

```diff
@@ -0,0 +1,119 @@
+#MySQL
+
+Allows to exchange data with the remote MySQL server.
+
+You can connect to some database on the MySQL server and perform `INSERT` and `SELECT` queries with tables.
+
+MySQL database engine cannot perform the following operations with tables:
```

Comment:
As far as I understand all MySQL tables are "visible" by default, if so it's worth to explicitly mention that. Then this list will make more sense. I'd probably still write those verbs as keywords in backticks.